### ISSUE TITLE
r8169 / 2.3LK-NAPI

### ISSUE THREAD

**Issue by KarelWintersky** [isc-projects/ethq#3]
```
# ethtool -i enp1s0
driver: r8169
version: 2.3LK-NAPI
firmware-version:
expansion-rom-version:
bus-info: 0000:01:00.0
supports-statistics: yes
supports-test: no
supports-eeprom-access: no
supports-register-dump: yes
supports-priv-flags: no
```

```
# ethtool -S enp1s0
NIC statistics:
     tx_packets: 0
     rx_packets: 0
     tx_errors: 0
     rx_errors: 0
     rx_missed: 0
     align_errors: 0
     tx_single_collisions: 0
     tx_multi_collisions: 0
     unicast: 0
     broadcast: 0
     multicast: 0
     tx_aborted: 0
     tx_underrun: 0
```
Thank you!

**Comment by KarelWintersky** [comment]
it is an onboard card for GIGABYTE GA-N3150N-D3V, mini-ITX motherboard

**Comment by raybellis** [comment]
Thanks for the info :)

However, there's only a single queue on this NIC, and there's apparently no `rx_bytes:` or `tx_bytes:` counters available either.   To further complicate matters, the `rx_packets:` and `tx_packets:` lines also appear in the `i40e` driver output, which gives me no way to distinguish those.

I'll keep this in mind next time I poke at the code, but my priority will be multi-queue NICs.

**Comment by raybellis** [comment]
Resolved with 0701b830a383e3a2802ce9e953f5f1a00f98f1e5

**Comment by raybellis** [comment]
NB: the code will just show "0" for bytes and Mbps for now.  I'm considering ways in which I might omit this information (or show "-" instead) for cards that don't support byte counters.
